module/data/actor/skill.mjs

```diff
@@ -0,0 +1,23 @@
+import { FormulaField } from "../fields.mjs";
+
+/**
+ * An embedded data structure for individual skill data.
+ *
+ * @property {number} value            Proficiency level creature has in this skill.
+ * @property {string} ability          Default ability used for this skill.
+ * @property {object} bonuses          Bonuses for this skill.
+ * @property {string} bonuses.check    Numeric or dice bonus to skill's check.
+ * @property {string} bonuses.passive  Numeric bonus to skill's passive check.
+ */
+export default class SkillData extends foundry.abstract.DataModel {
```

Comment:
Same comment here as with `AbilityData`. I would really prefer these to be more lightweight `SchemaField`.